doc/whats_new.rst

```diff
@@ -51,6 +51,10 @@ New features
      converts single output regressors to multi-ouput regressors by fitting
      one regressor per output. By `Tim Head`_.
 
+   - Generalization of :func:`model_selection._validation.cross_val_predict`.
+     One can pass method names such as `predict_proba` to be used in the cross
+     validation framework instead of the default `predict`. By `Ori Ziv`_ and `Sears Merritt`_.
```

Comment:
The names here are hyperlinked, but the hyperlinks are defined at the end of this file. Please add entries for both names.